### DISCUSSION TITLE
npx create-payload-app in current directory

### DISCUSSION THREAD

**Discussion by TomSmedley** [payloadcms/payload#1359] [Feature Requests & Ideas]
Hey everyone,

This is a request I made on Discord a few months back but never made a discussion for it. In my workflow I create a folder for a project then open that folder in Visual Code Studio, from there I use the Visual Code Studio terminal to install everything I need into that folder. For example for NextJS I do `npx create-next-app@latest .` which installs it to the current directory. 

Trying to do the same for PayloadCMS returns the error `The project directory './' already exists`, this means I have to create it inside a directory then move it back out to the project root. Not a massive thing, but just a helpful thing.

**Comment by denolfe** [comment]
@TomSmedley I just released a beta version of create-payload-app with this functionality. Give it a try with `npx create-payload-app@beta .` and let me know what you think.
